.travis.yml

```diff
@@ -34,7 +34,7 @@ before_script:
 script:
     - if [[ $DEFAULT = 1 ]]; then vendor/bin/phpunit --coverage-text --coverage-clover=coverage.clover; fi
     - if [[ $PHPSTAN = 1 ]]; then composer require --dev phpstan/phpstan:^0.8 && vendor/bin/phpstan analyse -c phpstan.neon -l 4 src; fi
-    - if [[ $PHPCS = 1 ]]; then composer require --dev squizlabs/php_codesniffer:~3.1.1 && vendor/bin/phpcs -p -s src; fi
+    - if [[ $PHPCS = 1 ]]; then phpcs -p --standard=vendor/cakephp/cakephp-codesniffer/CakePHP src/ tests/ app/; fi
```

Comment:
cant do `then phpcs -p `
should be `then vendor/bin/phpcs -p `

Reply:
also probs will complain about cakephp standard not being available

Reply:
Good point, I re-use the  composer command then.